[OKD] 'Fedora Security Hardening' hyperlink broken (#91125)
openshift/openshift-docs: modules/security-hardening-what.adoc

= Choosing what to harden in {op-system}
ifdef::openshift-origin[]

> The link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9#Security[Security] category in the Red Hat Enterprise Linux 9 guide describes how you should approach security for any {op-system-base} system.

Reviewer: i guess there is no Centos equivalent? cc @ausil

> The link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9#Security[Security] category in the Red Hat Enterprise Linux 9 guide describes how you should approach security for any {op-system-base} system.

Reviewer 2: I'd phrase this as "in Red Hat Enterprise Linux 9 documentation" instead of "in the Red Hat Enterprise Linux 9 guide".

To answer @Prashanth684: Well, the only up to date CentOS docs are for contributors (how to contribute to CentOS Stream plus some info about various SIGs), you can find them [here](https://docs.centos.org/). There is a Wiki, which even has a [page](https://wiki.centos.org/HowTos(2f)OS_Protection.html) that would be perfect - but the page is fairly old, and the entire wiki is now a read-only archive, so updates aren't going to happen.

> The link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9#Security[Security] category in the Red Hat Enterprise Linux 9 documentation describes how you should approach security for any {op-system-base} system.

Reviewer 3: ```suggestion
To know how to approach security for any {op-system-base} system, see the link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9#Security[Security] category in the Red{nbsp}Hat Enterprise Linux 9 documentation.
```
This is up to you.
I was thinking about `"what" --> "how"` and avoiding `should`.
